Automattic/jetpack
# Fix Intro Offer price description

Fixes the billing term text for 1201210512993648-as-1203748989040194

## Proposed changes:

* Fix the discounted price and details text for pricing cards that contains a product that we offer a trial (an intro offer for the first month).
<img width="457" alt="image" src="https://user-images.githubusercontent.com/5714212/213015382-52793532-9cb7-484a-ad1d-3f10803d8d0f.png">
<img width="655" alt="image" src="https://user-images.githubusercontent.com/5714212/213015494-81114a40-229d-474c-aa44-d1e8451165f0.png">


### Other information:


## Jetpack product discussion


p1673966558676559-slack-C02LK1W8T4Z


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Checkout this branch locally
* Run `jetpack build packages/backup` and `jetpack build packages/my-jetpack`
* Go to My Jetpack, then Add VaultPress Backup
* Make sure the right price is present ($1/€1)
* Make sure that the text says "for the first month, billed yearly" instead of "/month, billed yearly"
* Install VaultPress Backup
* Without a plan, go to `/wp-admin/admin.php?page=jetpack-backup`
* Confirm that the fix for both the price and details text is there

Note: if you use a currency other than USD or EUR, it should display the localized price, with the correct currency code.
